ocpmodels/trainers/ocp_trainer.py

```diff
@@ -581,7 +594,7 @@ def run_relaxations(self, split="val"):
             if check_traj_files(
                 batch, self.config["task"]["relax_opt"].get("traj_dir", None)
             ):
-                logging.info(f"Skipping batch: {batch.sid.tolist()}")
+                logging.info(f"Skipping batch: {list(batch.sid)}")
```

Comment:
minor - you'll need to do the if/else casting here as well otherwise tensors will be returned as a list of tensors.

Reply:
is that really necessary for logging though?

Reply:
Done!